src/ray/raylet/scheduling/cluster_task_manager.cc

```diff
@@ -56,6 +56,18 @@ ClusterTaskManager::ClusterTaskManager(
       metric_tasks_dispatched_(0),
       metric_tasks_spilled_(0) {}
 
+void ReplyCancelled(const std::shared_ptr<internal::Work> &work,
```

Comment:
Is there any other method that we can reuse? I believe we have other logic that is about cancelling lease request

Reply:
I believe this isn't a new method.